src/test/scala/chiselTests/LTLSpec.scala

```diff
@@ -183,11 +183,12 @@ class LTLSpec extends AnyFlatSpec with Matchers {
   it should "support property disable operation" in {
     val chirrtl = ChiselStage.emitCHIRRTL(new RawModule {
       val a, b = IO(Input(Bool()))
-      val p0: Property = a.disable(b)
+      val p0: Property = a.disable(b.asDisable)
     })
+    chirrtl should include("node _T = bits(b, 0, 0)")
```

Comment:
Q: is possible to give the disable better default name?

Reply:
In this case, the problem isn't really the disable, it's that `.asBool` even on a `Bool` will lead to this useless bit extraction node (eg. https://scastie.scala-lang.org/LFI1oxdlRSuTbj9cze1WlQ). I played around with fixing it, but that change should be a separate PR.

Reply:
Note that `bits(b, 0, 0)` on a 1-bit `b` will trivially be constant folded to `b` by firtool so it's really not an issue other than slightly ugly in the emitted .fir.